app/validators/representation_order_validator.rb

```diff
@@ -17,6 +17,9 @@ def validate_representation_order_date
     validate_presence(:representation_order_date, 'blank')
     validate_on_or_before(Date.today, :representation_order_date, 'in_future')
     validate_on_or_after(earliest_permitted[:date], :representation_order_date, earliest_permitted[:error])
+    if scheme_ten?
```

Comment:
Will this validation create any duplicate error messages for advocate interim claims (at least through the web flow) since [this](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/claim/advocate_interim_claim_validator.rb#L24) is already in place?

Reply:
Okay @lostie, I've run through the scenarios (in my head, no cukes needed!) and this won'r be an issue, either on the web or API.  On the web, the offence is added after this page and we only show the appropriate offences for the (already validated by this) rep order.  In the API, the claim includes the offence, so then we validate the rep order date against that.  

My next job is hooking up the Advocate Interim end point on the API.  I may need to tweak these two rules then...